library/Icingadb/Widget/ViewModeSwitcher.php

```diff
@@ -154,6 +154,9 @@ protected function assemble()
 
             if ($viewMode === $currentViewMode) {
                 $input->getAttributes()->add('checked', true);
+                $label->getAttributes()->add('title', t(sprintf('%s view active', ucfirst($viewMode))));
```

Comment:
Three issues regarding localisation:

a) gettext will parse the source code to extract messages, so translatable strings need to be directly wrapped by the translation function (i.e. `t('%s view active')`)
b) Passing a computed result to any translation function isn't possible. gettext extracts the arguments for such translation functions as `t()` and hence computed values won't be.. computed :smile: 
c) Translatable messages may of course contain placeholders for sprintf, but then it's usually a value that is localized differently (such as a date) or not localized at all (such as a number). This however is a name, and names are localizable, they're adjectives after all. So here we'd need three different messages, one for each mode.

Reply:
This is resolved? You've only applied a) and b), what about c)?